packages/one-app-bundler/webpack/module/webpack.client.js

```diff
@@ -58,7 +58,8 @@ module.exports = (babelEnv) => {
       },
       node: { module: 'empty', net: 'empty', fs: 'empty' },
       performance: {
-        maxAssetSize: configOptions.performanceBudget || 200e3,
+        maxAssetSize: configOptions.performanceBudget || 250e3,
+        maxEntrypointSize: configOptions.performanceBudget || 250e3,
```

Comment:
Would we ever want these to be different?

Reply:
For the defaults? What were you thinking?

Reply:
@Francois-Esquire meaning should I be able to set `maxAssetSize` and `maxEntrypointSize` independently?

Reply:
That can be done via custom webpack config